### DISCUSSION TITLE
Help with html source

### DISCUSSION THREAD

**Discussion by vladosam** [ImranR98/Obtainium#1123] [Q&A] [answered]
Hi, how can i add app in obtainium if the app is hosted as links on simple html page?
```
app-production-08082023.apk
app-production-22072022.apk
```
If I go on https://mydomain.com:11111 I can see links like above. How can I setup obtainium to update my app? Also, can i access site if it is protected with basic auth with user and password? Thanks.

**Comment by ImranR98** [comment, marked as answer]
https://github.com/ImranR98/Obtainium/issues/954#issuecomment-1745970526

Auth would be possible with custom headers but that has not been implemented yet.

**Reply by vladosam** [reply]
Thanks.
